advisories/github-reviewed/2022/04/GHSA-fwr7-v2mv-hh25/GHSA-fwr7-v2mv-hh25.json

```diff
@@ -25,11 +25,14 @@
               "introduced": "0"
             },
             {
-              "fixed": "3.2.2"
+              "fixed": "3.2.2,2.6.4"
             }
           ]
         }
-      ]
+      ],
+      "database_specific": {
```

Comment:
How did you create this field? Was it automatic or manual? It wasn't created for me :thinking: